qdax/environments/wrappers.py

```diff
@@ -0,0 +1,73 @@
+from typing import Dict
```

Comment:
Is this wrapper really linked to that particular PR? Is there no way to make a separate PR for this wrapper?

Reply:
Yes, this wrapper is really needed.

In summary, this wrapper used to be present in Brax, and we rely on some elements of it in 4 algorithms (I think those are: SAC, DIAYN, DADS and TD3), and their tests where all failing for that reason.

So what I did is copying the old wrapper of Brax here, and now all the tests pass.

This is a provisional solution to make it work in the same way as before.
Maybe we can improve and adapt our own code to the new structure of QDax, but I think that is for another time/PR.

Reply:
Ok! Thanks for your answer! Could you add this in the PR description?